os_net_config/cli.py

```diff
@@ -202,17 +199,24 @@ def main(argv=sys.argv, main_logger=None):
                               "system.")
             return 1
 
+    try:
```

Comment:
Please move this block after line #312. Provider needs to be initialised after purge provider, so that the migration sequence is taken care.

Reply:
Done